Module.php

```diff
@@ -144,8 +144,9 @@ public function bootstrap($app)
         $app->on(Application::EVENT_BEFORE_REQUEST, function () use ($app) {
             $app->getView()->on(View::EVENT_END_BODY, [$this, 'renderToolbar']);
         });
-
-        $app->getUrlManager()->addRules([
+        $urlManager = $app->getUrlManager();
+        $urlManager->ruleConfig['class'] = 'yii\web\UrlRule';
```

Comment:
@samdark @SilverFire 

I do not see your comments...

Forget my pr. thanks for you to handle this quetion.
:+1: